include/geos/operation/buffer/OffsetCurveSetBuilder.h

```diff
@@ -176,6 +174,21 @@ class GEOS_DLL OffsetCurveSetBuilder {
     OffsetCurveSetBuilder(const OffsetCurveSetBuilder& other) = delete;
     OffsetCurveSetBuilder& operator=(const OffsetCurveSetBuilder& rhs) = delete;
 
+    /**
+    * Computes orientation of a ring using a signed-area orientation test.
+    * For invalid (self-crossing) rings this ensures the largest enclosed area
+    * is taken to be the interior of the ring.
+    * This produces a more sensible result when
+    * used for repairing polygonal geometry via buffer-by-zero.
+    * For buffer  use the lower robustness of orientation-by-area
```

Comment:
```suggestion
    * For buffer, using the lower robustness of orientation-by-area
```
I guess this lower robustness method is algorithm::Orientation::isCCW() ?

Reply:
`algorithm::Orientation::isCCWArea`